BraveRewardsUI/User Wallet/VerifyUserWalletButton.swift

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import pop
+
+/// The button to place on the wallet header when the user hasn't currently
+/// connected with Uphold
+class VerifyUserWalletButton: UIControl {
```

Comment:
A lot of various buttons is added for this feature.

I wonder if it would be better to embed them in some helper class to not pollute the global namespace.
So it would be accessed like `UserWalletUI.VerifyUserWalletButton`

Reply:
Yeah they all have somewhat different UI so I just made them different classes in case the UI changes (as it has on desktop a few times). 

I could do namespaces but I don't see a point since they'll pollute Xcode's `Open Quickly` list regardless